.gitignore

```diff
@@ -1,7 +1,7 @@
-# Python
-__pycache__
+.mypy_cache
 .python-version
-
-# Terraform
 .terraform
+__pycache__
+terraform.tfstate
+terraform.tfstate.backup
```

Comment:
Nitpick, but I don't think these are sorted.

Reply:
I think these are sorted by punctuation first - what's our standard?
```
.mypy_cache
.python-version
.terraform
__pycache__
terraform.tfstate
terraform.tfstate.backup
*.tfvars
```

Reply:
Resolved in https://github.com/cisagov/assessment-data-import-terraform/pull/4/commits/162059b0b8d52f616be3e823f55e54e264183b5d

Reply:
I'd move the `*.tfvars` up with the punctuation stuff since we're alphabetizing these, but I'm not very worked up about it.